### ISSUE TITLE
"F811 redefinition of unused" will be not fixed

### ISSUE THREAD

**Issue by jedie** [astral-sh/ruff#9652]
ruff 0.1.14

e.g.:
```
from sys import version_info
from sys import version_info
from sys import version_info
from sys import version_info
from sys import version_info


print(version_info)
```
is valid for ruff and `ruff format` will not fix: **F811 redefinition of unused** but F ist active.

**Comment by charliermarsh** [comment]
This exists in preview mode. If you run `ruff check --fix --preview`, it will remove redefined imports.

(`ruff format` doesn't remove redefined imports, since the formatter intends not to change the syntax tree / meaning of the code.)

**Comment by jedie** [comment]
So this will be come in a later release?

**Comment by charliermarsh** [comment]
Yeah, it'll be enabled on stable in a later release. (Hopefully in the next few weeks.)

**Comment by jedie** [comment]
I use v0.1.14 and the screenshot is made from this version.

**Comment by charliermarsh** [comment]
Ah yeah. The rule exists in preview and on stable, but the automatic _fix_ is only available in preview right now. The docs don't reflect that specific nuance (that the rule exists, but the fix requires preview).

**Comment by jedie** [comment]
OK. Thanks.

**Comment by jedie** [comment]
Sorry... Even with `--preview` it's not fixed...
